src/style.css

```
@@ -1 +1,90 @@
-/* Styles go here */
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("./fonts/Roboto-Medium.ttf") format("truetype");
+  font-weight: 500;
+  font-style: normal;
+}
+
+body {
+  font-family: Roboto, serif;
+  font-size: 12px;
+}
+
+.header {
+  position: absolute;
+  left: 0;
+  top: 0;
+  margin: auto;
+  width: 1200px;
+  height: 60px;
```

```suggestion
  width: 100%;
```
do not hardcode height and width